pontoon/base/models.py

```diff
@@ -49,7 +49,7 @@ def _changed_translations_count(self, *args):
             Case(
                 When(translation_query, then=1), output_field=models.IntegerField(), default=0))
 
-    def with_translation_counts(self, start_date=None, limit=100):
+    def with_translation_counts(self, start_date=None, query_args=None, limit=100):
```

Comment:
`query_filters` might be a better name, `args` doesn't tell me what the argument is for. Also, it could use some documentation in the docstring.